x-pack/plugins/security_solution/server/lib/entity_analytics/entity_store/routes/init.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { buildSiemResponse } from '@kbn/lists-plugin/server/routes/utils';
+import { transformError } from '@kbn/securitysolution-es-utils';
+import { ENTITY_STORE_INIT_URL } from '../../../../../common/constants';
+import type { SecuritySolutionPluginRouter } from '../../../../types';
+export const entityStoreInitRoute = (router: SecuritySolutionPluginRouter) => {
+  router.versioned
+    .post({
+      access: 'internal',
+      path: ENTITY_STORE_INIT_URL,
+      options: {
+        tags: ['access:securitySolution'], // TODO entity store access `access:${APP_ID}-entity-analytics`
```

Comment:
when will we be able to implement this? Not too familiar with how these access tags work

Reply:
This is what we call a capability. Users can access the feature when they have the capability configured. 
Some capabilities can be disabled on stack management UI, others are only associated with the Product Line Item (PLI) on serverless. Entity analytics has a capability called 'entity-analytics` that controls access to all Entity analytics features.

We declare the entity analytics capability here: [/Users/machadoum/workspace/kibana/x-pack/packages/security-solution/features/src/security/app_feature_config.ts](https://github.com/elastic/kibana/blob/94284a105a7af055d35d0b758a448f4e76de9147/x-pack/packages/security-solution/features/src/security/app_feature_config.ts#L26)
Documentation about how PLIs are implemented: https://docs.google.com/document/d/1Ms8d8d_fbTTRHlBroEAKGNMNk3jFFgOAkVDRhqLxAPQ/edit

I don't know if Entity Store will be accessible by all users or only the ones that can access Entity Analytics. I will remove the comment since we might not have to implement it.